products/nodejs.md

```diff
@@ -63,7 +69,12 @@ releases:
     latest: "10.24.1"
     latestReleaseDate: 2021-04-06
     releaseDate: 2018-04-24
-
+-   releaseCycle: "8"
+    release: 2017-10-31
+    lts: true
```

Comment:
I think Web Archive has some of the older LTS start dates.

Reply:
Thanks for the hint, found: https://web.archive.org/web/20190317014458/https://nodejs.org/en/about/releases/